### DISCUSSION TITLE
Milvus Enterprise Manager licensing

### DISCUSSION THREAD

**Discussion by thinline72** [milvus-io/milvus#3794] [Q&A and General discussion] [answered]
Hi all,

Recently I've noticed that there is a cool UI tool for managing Milvus: https://zilliz.com/products/em/
It's very easy to install and use. But I'm not able to find any license related things. It also doesn't seem to be open-sourced.

So I'm wondering what are the restrictions on using Milvus Enterprise Manager? Can it be used for free including commercial usage similarly to Milvus?

Thanks!

**Comment by shiyu22** [comment, marked as answer]
Hi,

The current version of Milvus EM is of course free and you can use it however you like.

By the way, Milvus EM is free, but not open source.
